### DISCUSSION TITLE
Github Desktop ... Linux?

### DISCUSSION THREAD

**Discussion by ashbhay** [community/community#112259] [New to GitHub]
Is there a way to install the desktop on a linux os ?

**Comment by kuciapakrystian** [comment]
Yes, you can install a desktop environment on a Linux operating system if it doesn't already have one. Here's how you can do it:

Choose a Desktop Environment: There are several desktop environments available for Linux, such as GNOME, KDE Plasma, Xfce, LXQt, and many others. Choose the one that suits your preferences and system resources.

Install the Desktop Environment: Depending on your Linux distribution, you can usually install a desktop environment using the package manager. Here are the commands for some popular distributions:

Ubuntu/Debian: sudo apt-get install ubuntu-desktop (for GNOME), sudo apt-get install kubuntu-desktop (for KDE Plasma), sudo apt-get install xubuntu-desktop (for Xfce), etc.

Fedora: sudo dnf install @gnome-desktop (for GNOME), sudo dnf install @kde-desktop (for KDE Plasma), sudo dnf install @xfce-desktop (for Xfce), etc.

CentOS/RHEL: Desktop environments are not commonly installed by default, but you can add repositories or install individual packages. For example, you can install GNOME with sudo yum groupinstall "GNOME Desktop".

Configure Display Manager (Optional): If you're installing a desktop environment on a system that previously didn't have one, you may need to configure the display manager to start the desktop environment automatically. This varies depending on the display manager you're using (e.g., GDM, LightDM, SDDM).

Reboot: After installing the desktop environment, reboot your system. You should then be able to log in to the newly installed desktop environment from the login screen.

Keep in mind that installing a desktop environment can consume system resources, so make sure your system meets the requirements of the chosen desktop environment. Additionally, having multiple desktop environments installed on the same system can sometimes lead to conflicts, so it's generally best to stick with one unless you have specific reasons for needing multiple environments.

**Reply by ghostinhershell** [reply]
Hi @kuciapakrystian, 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by ghost** [reply]
hi @kuciapakrystian
You can download the github windows version and install it by wine package/tool which is available on most linux distros
this package lets you to install/run windows executable file

**Comment by hafiz-muhammad** [comment]
You can download GitHub Desktop for Linux from Flathub by following this link: [https://flathub.org/apps/io.github.shiftey.Desktop](https://flathub.org/apps/io.github.shiftey.Desktop)

**Comment by Orionisus** [comment]
Cool

**Comment by Aryosetowmn** [comment]
Hi @ashbhay 

I usually use this method 👇

1. Check latest version: https://github.com/shiftkey/desktop
2. Download: https://github.com/shiftkey/desktop/releases/download/release-3.1.1-linux1/GitHubDesktop-linux-3.1.1-linux1.deb
3. Go to the file directory in `terminal`
4. type this command `sudo dpkg -i filename.deb`
5. enter your linux password
6. Wait Till the process is finished
7. DONE 🙂👍

**Comment by TushantMan** [comment]
Hi @ashbhay 

You can use this  Electron-based GitHub app for Linux: https://github.com/shiftkey/desktop?tab=readme-ov-file#installation-via-package-manager

Or by following the steps below:
Step 1 :
Use the wget command followed by the URL of the .deb file of the GitHub desktop from the shiftkey Github repo. Shifykey is maintained by Brendan Forster one of the staff members of Github.

sudo wget https://github.com/shiftkey/desktop/releases/download/release-3.1.1-linux1/GitHubDesktop-linux-3.1.1-linux1.deb

Step 2 :
Install gdebi using the following command which helps you to install local deb packages, resolving and installing its dependencies.

sudo apt-get install gdebi-core

Step 3 :
After installing gdebi, using gdebi install the GitHubDesktop client using the following command.

sudo gdebi GitHubDesktop-linux-2.9.0-linux2.deb
